mage/sh/lib.go

```diff
@@ -42,14 +48,22 @@ func RunPanic(cmd string, args ...string) {
 
 // Run command and print any output to stdout/stderr
 func RunV(cmd string, args ...string) error {
-	_, err := sh.Exec(nil, os.Stdout, os.Stderr, cmd, args...)
+	return RunVWithEnv(nil, cmd, args...)
+}
+
+func RunVWithEnv(env map[string]string, cmd string, args ...string) error {
+	_, err := sh.Exec(env, os.Stdout, os.Stderr, cmd, args...)
 	return err
 }
 
 // Run command and print any output to stdout/stderr
 // Will automatically panic on error
 func RunVPanic(cmd string, args ...string) {
-	err := RunV(cmd, args...)
+	RunVPanicWithEnv(nil, cmd, args...)
+}
+
+func RunVPanicWithEnv(env map[string]string, cmd string, args ...string) {
```

Comment:
There's gotta be some sort of a "panic'd running with envy" joke laying around here somewhere...